### ISSUE TITLE
Bug: High memory usage of tracim backend webapp

### ISSUE THREAD

**Issue by grignards** [tracim/tracim#4163]
## Description and expectations

The memory usage of the main tracim web application seems to be growing for unexplained reasons.
We should investigate to know whether this memory behavior is due to our code or simply to the python memory allocation mechanism.

### How to reproduce

1. Start a tracim instance
2. Use it for a while
3. Memory usage grows over time

### Version information

- Tracim version: 3.4.3

## Screenshots

________________________________________________

**Comment by PhilippeAccorsi** [comment]
I have uploaded a same file 5 times in one folder (with different name. File size is 240Mo), and play de video for each file. I used the video progress bar also to load data in ram on server side. The memory consumption increase for each video I played. After test, memory increasy by 600Mo and not released.

I think each video stay in buffer.

**Comment by inkhey** [comment]
@grignards @PhilippeAccorsi I'm curious about the way you are running tracim and monitoring the memory usage for this.

**Comment by PhilippeAccorsi** [comment]
I use htop @inkhey and run tracim with docker image (tracim use elasticsearch, collabora, reply by email, postgres database).

**Comment by lebouquetin** [comment]
@grignards @inkhey do you have an idea of the origin of this issue?

**Comment by grignards** [comment]
No, we investigated a bit with @PhilippeAccorsi and found out the problem with files upload/download but we didn't spend more time investigating the cause.

**Comment by lebouquetin** [comment]
Ok, thanks.
